libs/utils/website/src/lib/prerender-teaser-paths.tsx

```diff
@@ -52,11 +52,16 @@ export const getArticlePathsBasedOnPage =
     })
 
     const cache = Object.values(client.cache.extract())
-    const articleSlugs = []
+    const articleSlugs = [] as {slug: string; tag?: string}[]
 
     for (const storeObj of cache) {
       if (storeObj?.__typename === 'Article' && !(storeObj as ApiV1.Article).peeredArticleURL) {
-        articleSlugs.push((storeObj as ApiV1.Article).slug)
+        const article = storeObj as ApiV1.Article
+
+        articleSlugs.push({
+          slug: article.slug,
+          tag: article.tags.find(tag => tag.main)?.tag ?? undefined
```

Comment:
<img width="1417" alt="image" src="https://github.com/wepublish/wepublish/assets/1131901/33fcc135-d4d4-40ba-b8bd-f55b106f1569">

checked it still properly pre-renders even if only slug is present (for example for tsri)